.github/workflows/check-excluded-packages.yml

```diff
@@ -50,15 +50,16 @@ jobs:
           packages="${packages//$'\n'/'%0A'}"
           targets="${targets//$'\n'/'%0A'}"
 
-          echo "::set-output name=packages::$packages"
-          echo "::set-output name=targets::$targets"
+          echo "packages=$packages" >> $GITHUB_OUTPUT
+          echo "targets=$targets" >> $GITHUB_OUTPUT
 
       - name: Check excluded packages
         env:
           RUSTC_WRAPPER: ""
           RUSTC_WORKSPACE_WRAPPER: sccache
         run: |
           packages="${{ steps.format_output.outputs.packages }}"
+          packages="${packages//'%0A'/$'\n'}"
```

Comment:
I wonder how it did wor before that

Reply:
@mikogs found this: https://github.com/orgs/community/discussions/26288 - apparently it was a feature

I wonder why it does not work anymore...